configparser.y

```diff
@@ -475,16 +482,18 @@ server_ranges:
         }
         assert(first >= 0);
         assert(last >= 0);
+        opt->next = NULL;
         opt->first = (int)first;
         opt->last = (int)last;
-        if(tail) {
-          tail->next = opt;
-          tail = opt;
+        if(servers) {
+          servers = servers->next = opt;
         } else {
-          $$ = tail = opt;
+          servers = cfg_parser->ip->servers = opt;
```

Comment:
It looks like the assignment to `servers` is unused.  The variable is declared on line 471 and goes out of scope below.

Reply:
The notation maybe a little icky, but the first time around servers is assigned `cfg_parser->ip->servers` all consecutive operations assign the `next` server to servers. So `servers` is indeed a throwaway variable, but no memory is leaked. The `ip`, of which `servers` is a member is assigned in another block.